website/pages/docs/secrets/transform/tokenization.mdx

```diff
@@ -8,96 +8,96 @@ description: >-
 
 # Tokenization Transform
 
-Not to be confused with Vault tokens, [Tokenization](transform/tokenization) exchanges a 
-sensitive value for an unrelated value called a *token*. The original sensitive 
-value cannot be recovered from a token alone, they are irreversible.    Instead, unlike format preserving encryption, tokenization is stateful.  To 
-decode the original value, the token must be submitted to Vault where it is 
-retrieved from a cryptographic mapping in storage.  
+Not to be confused with Vault tokens, [Tokenization](transform/tokenization) exchanges a
+sensitive value for an unrelated value called a *token*. The original sensitive
+value cannot be recovered from a token alone, they are irreversible.    Instead, unlike format preserving encryption, tokenization is stateful.  To
```

Comment:
Is the number of spaces in between sentences correct in here?

Reply:
I didn't touch this section at all.  I think `git commit` did something.

Reply:
The [Netlify deploy preview](https://deploy-preview-10411--vault-www.netlify.app/docs/secrets/transform/tokenization) looks fine, so I think it's good to go now. 🙏